lib/matplotlib/blocking_input.py

```diff
@@ -40,8 +39,6 @@ class BlockingInput(object):
     """
     def __init__(self, fig, eventslist=()):
         self.fig = fig
-        if not is_sequence_of_strings(eventslist):
-            raise ValueError("Requires a sequence of event name strings")
```

Comment:
Because it's not as if the check was really useful anyways -- one can always pass a sequence of invalid event names and it'll be just as bad as of one passed in a sequence of nonstrings.